src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
>We still don't want this test to be innerloop because in best cases it takes 40 seconds. And that is a long time for a single unit test.

Agreed. And if it's Outerloop anyways, then hitting a long timeout on failure isn't as big of a deal. It's of course not ideal that it could go so long, but at least we won't have it fail intermittently unless there's something actually wrong or the test server is down